ibm_mq/datadog_checks/ibm_mq/data/conf.yaml.example

```diff
@@ -1,142 +1,163 @@
+## All options defined here are available to all instances.
+#
 init_config:
 
+    ## @param service - string - optional
+    ## Attach the tag `service:<SERVICE>` to every metric, event, and service check emitted by this integration.
+    ##
+    ## Additionally, this sets the default `service` for every log source.
+    #
+    # service: <SERVICE>
+
+## Every instance is scheduled independent of the others.
+#
 instances:
+
     ## @param channel - string - required
     ## The IBMMQ Channel to connect to.
     #
-  - channel: 'DEV.ADMIN.SVRCONN'
+  - channel: DEV.ADMIN.SVRCONN
 
     ## @param queue_manager - string - required
     ## The name of the queue manager.
     #
-    queue_manager: 'datadog'
+    queue_manager: datadog
 
-    ## @param host - string - required - default: localhost
+    ## @param host - string - required
     ## The host IBM MQ is running on.
     #
-    host: 'localhost'
+    host: localhost
 
-    ## @param port - string - required - default: 1414
+    ## @param port - string - required
     ## The port IBM MQ is listening on.
     #
     port: '1414'
 
-    ## @param username - string - optional
     ## <USERNAME> of the IBMMQ Channel to connect to.
     #
-    # username: '<USERNAME>'
+    # username: {}
 
-    ## @param password - string - optional
     ## <PASSWORD> of the IBMMQ Channel to connect to.
     #
-    # password: '<PASSWORD>'
+    # password: {}
 
-    ## @param queues - list of string - optional
+    ## @param queues - list of strings - optional
     ## The IBMMQ queues to get data from (no queue metrics will be sent, but the check will run without it).
     #
     # queues:
-    #   - '<QUEUE_NAME>'
+    #   - <QUEUE_NAME>
 
-    ## @param queue_patterns - list of string - optional
+    ## @param queue_patterns - list of strings - optional
     ## Collect from queues that match a MQ pattern
     #
     # queue_patterns:
-    #   - 'DEV.*'          # match queues starting with "DEV."
-    #   - 'SYSTEM.*'       # match queues starting with "SYSTEM."
+    #   - DEV.*
```

Comment:
Is there a way to keep comments?

Reply:
No, just add to description if necessary